static/app/utils/replays/replayReader.tsx

```diff
@@ -71,6 +89,44 @@ export default class ReplayReader {
     replayRecord,
     errors,
   }: RequiredNotNull<ReplayReaderParams>) {
+    const {breadcrumbFrames, optionFrame, rrwebFrames, spanFrames} =
+      hydrateFrames(attachments);
+
+    if (localStorageWrapper.getItem('REPLAY-BACKEND-TIMESTAMPS') !== '1') {
+      // TODO(replays): We should get correct timestamps from the backend instead
+      // of having to fix them up here.
```

Comment:
Can we track how often these two timestamps are still mismatched? It's more of a SDK issue than backend.

Reply:
ya good call. i'll throw it into amplitude so we can see the good vs. bad counts and maybe be able to group by unique replayid